sdk/tests/conformance/glsl/misc/00_test_list.txt

```diff
@@ -5,6 +5,7 @@ embedded-struct-definitions-forbidden.html
 --min-version 1.0.4 empty-declaration.html
 empty_main.vert.html
 --min-version 1.0.3 expression-list-in-declarator-initializer.html
+fragcolor-fragdata-invariant.html
```

Comment:
Could you please add --min-version 1.0.4 for bookkeeping purposes?

Reply:
Of course, done.